release-notes.md

```diff
@@ -86,13 +86,13 @@ Byte Buddy release notes
 - Add method to get possibly known class file version of TypeDescription.
 - Correctly consider generic array type when computing erasure.
 
-### 9. Mars 2021: version 1.10.22
+### 9. March 2021: version 1.10.22
 
 - Refactor `JavaConstant` API and fix handle resolution.
 - Adjust bootstrap method validation to account for possible dynamic constants.
 - Make class loaders parallel capable.
 - Correct code for attaching to Windows VMs with 32 bit.
-- Allow configuration for Gradle plugin to consider class path as incremental. 
+- Allow configuration for Gradle plugin to consider class path as incremental.
```

Comment:
My editor is set to trim trailing whitespace on file save, so a bunch of these slipped in as well. Can move them to a separate PR if you so prefer.